ee/initialize.go

```diff
@@ -48,6 +48,9 @@ func setControllerLimits() {
 	logic.Clients_Limit = Limits.Clients
 	logic.Free_Tier = Limits.FreeTier
 	servercfg.Is_EE = true
+	if logic.Free_Tier {
+		logic.Networks_Limit = 3
```

Comment:
Should this be a constant

Reply:
It's only used once, generally constants are made if values are necessary if used more than once.